lumicks/pylake/correlated_stack.py

```diff
@@ -299,11 +307,18 @@ def plot_correlated(
         plot_correlated(
             channel_slice,
             frame_timestamps,
-            lambda frame_idx: self._get_frame(frame_idx)._get_plot_data(channel),
+            lambda frame_idx: self._get_frame(frame_idx)._get_plot_data(
+                channel, adjustment=adjustment
+            ),
```

Comment:
maybe define this before the function call?

Reply:
Done and done.